SpatialGDK/Source/SpatialGDKFunctionalTests/Public/TestMaps/Spatial2WorkerSmallInterestMap.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "TestMaps/GeneratedTestMap.h"
+#include "Spatial2WorkerSmallInterestMap.generated.h"
+
+/**
+ * This map is a simple 2-server-worker map, where both workers see only a tiny bit (150 units) outside of their authoritative zone.
+ */
+// meta comment ^ not sure how useful it is to include the interest radius here, as this is basically duplicating information... could just
```

Comment:
Agreed; you can also add a namespace/struct with static values that would show the radius:
```
namespace Spatial2WorkerSmallInterestValues
{
constexpr static float InterestRadius = 150.0f;
}
```

Reply:
Hm, could, but this is currently not used by anything.
Would it be exposed by the strategy itself or this map?